docs/_source/guides/llms/conceptual_guides/sft.md

```diff
@@ -118,10 +119,24 @@ records = [
 ]
 
 dataset.add_records(records)
+```
+
+::::{tab-set}
 
-# This publishes the dataset and pushes the records into Argilla
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+# This publishes the dataset with the records into Argilla and returns the dataset in Argilla
+remote_dataset = dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")
```

Comment:
Same as above `remote_dataset` is not used later. Maybe we can just `push_to_argilla()` in this case?